doc/_includes/tutorials/common/samples/copy-samples.md

```diff
@@ -40,15 +46,40 @@ then select the checkbox "Remove samples from current project (move samples)"
 
 ### Locking Samples (ONLY FOR SHARING NOT MOVING)
 
-If you are sharing samples and you do not want them to be modifiable in the destination project, select the checkbox "Prevent modification of samples in target project (only when copying samples)"
+If you are sharing samples and you do not want them to be modifiable in the destination project, select the checkbox "
+Prevent modification of samples in target project (only when copying samples)"
 
 ![Share samples locks checkbox.]({{ site.baseurl }}/images/tutorials/common/samples/share-lock-checkbox.png)
 
 **NOTE: Both checkboxes cannot be selected at the same time.**
 
+### Metadata Restrictions
+
+![Share samples metadata restrictions.]({{ site.baseurl }}/images/tutorials/common/samples/share-metadata.png)
+
+Samples are shared/moved with all their corresponding metadata. This step allows you to review the fields that are
+included with the sample as well as review the metadata restriction level that will used on the destination project.
+
+If the metadata field does not exist in it will be set to the current level in the current project.
+
+![Share samples metadata restrictions.]({{ site.baseurl }}/images/tutorials/common/samples/share-metadata-current.png)
+
+In this example, metadata field `secondaryPfge` has a restrictin level of `Level 4`, the highest setting,
```

Comment:
Typo in `restriction` and change `highest setting` to `highest level`

Reply:
updated in Updated in 0e8f9a2

Reply:
updated in Updated in 0e8f9a2